charts/partials/templates/_proxy.tpl

```diff
@@ -159,7 +159,8 @@ livenessProbe:
   httpGet:
     path: /live
     port: {{.Values.proxy.ports.admin}}
-  initialDelaySeconds: 10
+  initialDelaySeconds: {{.Values.proxy.livenessProbe.initialDelaySeconds | default 10}}
+  timeoutSeconds: {{.Values.proxy.livenessProbe.timeoutSeconds | default 1}}
```

Comment:
I would get rid of the `default` statements, as we already have defaults in the `values.yaml` file, and to be consistent with the rest of the file (see `startupProbe`).

Reply:
yeah I did not want to touch the impl but I agree.